store/listings/management/commands/fetch_listings.py

```diff
@@ -37,6 +40,17 @@ def handle(self, *args, **options):
                 'filename': 'cotton_eur.txt'
             },
         ]
-
         for listings in listings_data:
-            save_to_file(listings=listings)
+            for listing in listings['data']:
+                listing_object = Listing.objects.create(
+                    title=listing.get('title'),
+                    description=listing.get('description'),
+                    price=listing.get('price'),
+                    currency=listing.get('currency'),
```

Comment:
Your `Listing` model has field `currency_code` not `currency` - line 49 will lead to exception

Reply:
API response also has `currency_code` key, not `currency`